src/app/_services/image-group.service.ts

```diff
@@ -17,6 +18,9 @@ import { ImageGroup, ImageGroupDescription } from './../shared'
  *
  */
 
+// The max number of images the user is able to bulk export.
+const MAX_DOWNLOAD_AMOUNT = 150;
```

Comment:
150 is the max number of images that can be exported/downloaded at one time. This restriction was actually only enforced on the service side. For example, if an image group contained say 300 images, we on the front end would pass ALL 300 image ids to the service, but the service would only return 150 in the zip/pptx. 

Since we need to log only the 150 images actually downloaded, we needed to move this restriction to the front end.

Reply:
🤔 So if the service chances to 100, it'll cause a problem? Is it possible to make the service the source of truth?

Reply:
For sure. TBH we are kinda of stuck right now. The `artstor-download-service` is currently logging this event. And it works because they know exactly which images are being downloaded. They have been pushing back on us to convert this log to the front end. But we really don't know for certain what actually gets downloaded. Maybe we should be getting a response from them that contains a complete list of all the ids. Or we can make an argument to continue logging on the service side.

Reply:
I think a conversation with them is warranted. The list of IDs downloaded is interesting. Talking offline, I also brought up the idea of returning a 400 response if the number of images to download is over the limit.

Reply:
Meeting with STORK at 10am on Monday

Reply:
After talking with STORK they agreed the solution here is to continue logging this on the front end. I went ahead and removed the download limit and logging code from this PR. Check out this commit: https://github.com/ithaka/aiw-ui/pull/1183/commits/e92625af6ff09374706affbd68f5729916c22e76